f5-openstack-agent-dist/Docker/ubuntu/install_test/Dockerfile.trusty

```diff
@@ -4,7 +4,10 @@ FROM ubuntu:trusty
 RUN apt-get update -y && apt-get install software-properties-common -y
 RUN apt-add-repository cloud-archive:liberty
```

Comment:
This should work.  Would you remove the cloud-archive:mitaka and run this test with cloud-archive:liberty